examples/mic_mim.py

```diff
@@ -0,0 +1,410 @@
+# patterns shows that the greatest connectivity between the left and right
+# hemispheres occurs for the posterolateral parietal regions of the left
+# hemisphere, and the medial central regions of the right hemisphere.
+#
+# Using the signs of the values, we can infer the existence of a parietal
+# dipole source in the left hemisphere which may account for the connectivity
+# contributions seen for the posterolateral parietal regions and frontomedial
+# areas (represented on the plot as a green line).
+
+# %%
+
+# compute average of patterns in desired frequency range
+fband = [13, 18]
+fband_idx = [mic.freqs.index(freq) for freq in fband]
+
+patterns = np.array(mic.attrs["patterns"])
+seed_pattern = np.mean(patterns[0, 0, :, fband_idx[0]:fband_idx[1] + 1],
+                       axis=1)
+target_pattern = np.mean(patterns[1, 0, :, fband_idx[0]:fband_idx[1] + 1],
+                         axis=1)
+
+# store the patterns for plotting
+seed_info = epochs.copy().pick(seed_names).info
+target_info = epochs.copy().pick(target_names).info
+seed_pattern = EvokedArray(seed_pattern[:, np.newaxis], seed_info)
+target_pattern = EvokedArray(target_pattern[:, np.newaxis], target_info)
+
+# plot the patterns
+fig, axes = plt.subplots(1, 4)
+seed_pattern.plot_topomap(
+    times=0, sensors='m.', units=dict(mag='A.U.'), cbar_fmt='%.1E',
+    axes=axes[0:2], time_format='', show=False)
+target_pattern.plot_topomap(
+    times=0, sensors='m.', units=dict(mag='A.U.'), cbar_fmt='%.1E',
+    axes=axes[2:], time_format='', show=False)
+axes[0].set_position((0.1, 0.1, 0.35, 0.7))
+axes[1].set_position((0.4, 0.3, 0.02, 0.3))
+axes[2].set_position((0.5, 0.1, 0.35, 0.7))
+axes[3].set_position((0.9, 0.3, 0.02, 0.3))
+axes[0].set_title('Seed spatial pattern\n13-18 Hz')
+axes[2].set_title('Target spatial pattern\n13-18 Hz')
+
+# plot the left hemisphere dipole example
+axes[0].plot(
+    [-0.07, -0.035], [-0.03, -0.055], color='lime', linewidth=2,
+    path_effects=[pe.Stroke(linewidth=4, foreground='k'), pe.Normal()])
+
+plt.show()
+
+
+###############################################################################
+# Multivariate interaction measure (MIM)
+# --------------------------------------
+#
+# Although it can be useful to analyse the single, largest connectivity
+# component with MIC, multiple such components exist and can be examined with
+# MIM. MIM can be thought of as an average of all connectivity components
+# between the seeds and targets, and can be useful for an exploration of all
+# available components. It is unnecessary to use the spatial filters of each
+# component explicitly, and instead the desired result can be achieved from
+# :math:`E` alone (Eq. 14 of :footcite:`EwaldEtAl2012`):
+#
+# :math:`MIM=tr(\boldsymbol{EE}^T)`,
+#
+# where again the frequency dependence is omitted. Unlike MIC, MIM is
+# positive-valued, and is not bound below 1. Without normalisation, MIM can be
```

Comment:
What does "not bound below 1" mean?

Reply:
MIM can be > 1. Rewritten to "Unlike MIC, MIM is positive-valued and can be > 1."